idea/src/org/jetbrains/kotlin/idea/quickfix/ChangeCallableReturnTypeFix.kt

```diff
@@ -74,8 +73,8 @@ abstract class ChangeCallableReturnTypeFix(
         val element = element!!
         if (element.name == null) return null
         val container = element.unsafeResolveToDescriptor().containingDeclaration as? ClassDescriptor
-        val containerName = container?.name?.takeUnless { it.isSpecial }
-        return StringPresentation.functionOrConstructorParameterPresentation(element, containerName)
+        val containerName = container?.name?.takeUnless { it.isSpecial }?.asString()
```

Comment:
Nit: I saw you added something like `identifierIfNotSpecial`?

Reply:
Yeah, that's also in #4338.

Reply:
Ditto. 🤦